src/python/WMComponent/JobSubmitter/JobSubmitterPoller.py

```diff
@@ -363,6 +362,11 @@ def refreshCache(self):
                 numberOfCores = getattr(baggage, "numberOfCores", 1)
             loadedJob['numberOfCores'] = numberOfCores
 
+            # check if job should be submitted as highIO job
+            highIOjob = False
+            if newJob['type'] in ["Repack", "Merge", "Cleanup", "LogCollect"]:
```

Comment:
Yes, in the end I believe it's a case by case. Super big workflows would create a IObound harvest job. Small ones wouldn't. Anyways it's merged already LoL